packages/kit/test/apps/basics/src/routes/prerendering/env/prerendered/+page.svelte

```diff
@@ -0,0 +1 @@
+<h2>prerendered</h2>
```

Comment:
Is this missing a `+page.js` with `export const prerender = true`? Else this isn't actually prerendered.